echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/eventhandlers/DockerEventHandler.java

```diff
@@ -87,7 +87,10 @@ public boolean isSuccessfulTriggerEvent(DockerEvent dockerEvent) {
   @Override
   protected Function<Trigger, Trigger> buildTrigger(DockerEvent dockerEvent) {
     return trigger ->
-        trigger.atTag(dockerEvent.getContent().getTag()).withEventId(dockerEvent.getEventId());
+        trigger
+            .atTag(dockerEvent.getContent().getTag())
```

Comment:
It's small but can we make a new `atTag(String tag, String digest)`? We're building 3 trigger objects here (tag -> digest -> event). I'd even prefer a single call but I'd settle for two in the name of consistency :)